src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetHelper.cs

```diff
@@ -284,6 +284,15 @@ private static bool DetermineIsMonoRuntime(string dotnetRoot)
         return File.Exists(Path.Combine(sharedFramework, "mono-gc.h"));
     }
 
+    public static bool ShouldPublishComplex() {
+        if (string.Equals(Config.TargetArchitecture,"ppc64le") || string.Equals(Config.TargetArchitecture,"s390x"))
+        {
+            return false;
+        }
+        else
+            return true;
```

Comment:
Nit: inconsistent brace style with it.
```suggestion
    {
            return true;
    }
```

This could be simplified this to use an expression bodied method.

```
public static bool ShouldPublishComplex() =>
    string.Equals(Config.TargetArchitecture,"ppc64le") || string.Equals(Config.TargetArchitecture,"s390x"); 

```